experiments/kripke/experiment.py

```diff
@@ -12,9 +12,15 @@
 from benchpark.scaling import StrongScaling
 from benchpark.scaling import WeakScaling
 from benchpark.scaling import ThroughputScaling
+from benchpark.domaindecomposition import GlobalDomains
 from benchpark.caliper import Caliper
 
 
+@GlobalDomains(
+strong_scaling_policy="default",
+weak_scaling_policy="default",
+throughput_scaling_policy="default",
+)
```

Comment:
@pearce8 @scheibelp Implementation of global scaling policies lib/benchpark/domaindecomposition.py::GlobalDomains